docs/apache-airflow-providers-amazon/operators/eks.rst

```diff
@@ -0,0 +1,265 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Amazon Elastic Kubernetes Service (EKS) Operators
+=================================================
+
+`Amazon Elastic Kubernetes Service (Amazon EKS) <https://aws.amazon.com/eks/>`__  is a managed service
+that makes it easy for you to run Kubernetes on AWS without needing to stand up or maintain your own
+Kubernetes control plane. Kubernetes is an open-source system for automating the deployment, scaling,
+and management of containerized applications.
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
+------------------
+
+.. include:: _partials/prerequisite_tasks.rst
+
+Overview
+--------
+
+Airflow to Amazon Elastic Kubernetes Service (EKS) integration provides Operators to create and
+interact with the EKS clusters and compute infrastructure.
+
+ - :class:`~airflow.providers.amazon.aws.operators.eks`
+
+4 example_dags are provided which showcase these operators in action.
+
+ - ``example_eks_create_cluster.py``
+ - ``example_eks_create_cluster_with_nodegroup.py``
+ - ``example_eks_create_nodegroup.py``
+ - ``example_eks_pod_operator.py``
+
+
+.. _howto/operator:EKSCreateClusterOperator:
+
+Creating Amazon EKS Clusters
+----------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_eks_create_cluster.py`` uses ``EKSCreateClusterOperator`` to create an Amazon
```

Comment:
I've mentioned this before, but we shouldn't be covering the individual sample files.
https://github.com/apache/airflow/pull/16571#discussion_r659253383
If you are using the exampleinclude directive, each code snippet has a button that allows you to go to the source code.
<img width="931" alt="Screenshot 2021-06-30 at 01 41 49" src="https://user-images.githubusercontent.com/12058428/123881125-5f9c1480-d944-11eb-8fb4-1b07cdb82e83.png">
A similar approach is used by other organizations:
https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/examples-s3-objects.html
https://cloud.google.com/storage/docs/reporting-changes#storage_delete_bucket_notification-cpp
<img width="905" alt="Screenshot 2021-06-30 at 01 53 06" src="https://user-images.githubusercontent.com/12058428/123881912-f3221500-d945-11eb-9de1-717deaf9a7d5.png">

Reply:
I'm sorry, I don't quite follow what you are looking for here.  The existing doc files are all over the place as far as how detailed it should be, and I was just trying to be thorough.  Which of the doc files is our golden standard?  If you can show me an existing Airflow doc file that you like, I'll emulate that one.

Reply:
I once wrote a document on how to write documentation for operators, but now it has been deleted and I don't think I have any copy.

You can be inspired by the guides below.
http://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/natural_language.html 
http://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/datacatalog.html
http://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/mlengine.html

If you want to add something to the guides, feel free to describe it, but I'm not sure if filenames just needs to be added.

Reply:
Should be addressed with https://github.com/apache/airflow/pull/16571/commits/5743f3469c8257948b9b73e2a7b2bdafb2a1d832